apps/etterlatte-saksbehandling-ui/client/src/components/behandling/brevutfall/BrevutfallSkjema.tsx

```diff
@@ -66,186 +92,154 @@ export const BrevutfallSkjema = (props: {
-        <RadioGroup
-          legend={
-            <HelpTextWrapper>
-              Skal det etterbetales?
-              <HelpText strategy="fixed">
-                Velg ja hvis ytelsen er innvilget tilbake i tid og det blir utbetalt mer enn ett månedsbeløp. Da skal du
-                registrere perioden fra innvilgelsesmåned til og med måneden som er klar for utbetaling. Vedlegg om
-                etterbetaling skal da bli med i brevet.
-              </HelpText>
-            </HelpTextWrapper>
-          }
-          className="radioGroup"
-          value={harEtterbetaling}
-          onChange={(event) => {
-            const svar = event as HarEtterbetaling
-            setHarEtterbetaling(svar)
-            if (svar === HarEtterbetaling.NEI) {
-              setBrevutfallOgEtterbetaling({ ...brevutfallOgEtterbetaling, etterbetaling: undefined })
-            }
-          }}
-        >
-          <Radio size="small" value={HarEtterbetaling.JA}>
-            Ja
-          </Radio>
-          <Radio size="small" value={HarEtterbetaling.NEI}>
-            Nei
-          </Radio>
-        </RadioGroup>
-
-        {harEtterbetaling == HarEtterbetaling.JA && (
-          <HStack gap="4">
-            <MaanedVelger
-              fromDate={new Date(behandling.virkningstidspunkt?.dato ?? new Date())}
-              toDate={new Date()}
-              value={
-                brevutfallOgEtterbetaling.etterbetaling?.datoFom
-                  ? new Date(brevutfallOgEtterbetaling.etterbetaling?.datoFom)
-                  : undefined
-              }
-              onChange={(date) =>
-                setBrevutfallOgEtterbetaling({
-                  ...brevutfallOgEtterbetaling,
-                  etterbetaling: {
-                    ...brevutfallOgEtterbetaling.etterbetaling,
-                    datoFom: date ? formatISO(startOfMonth(date), { representation: 'date' }) : undefined,
-                  },
-                })
-              }
-              label="Fra og med"
-            />
-            <MaanedVelger
-              fromDate={new Date(behandling.virkningstidspunkt?.dato ?? new Date())}
-              toDate={add(new Date(), { months: 1 })}
-              value={
-                brevutfallOgEtterbetaling.etterbetaling?.datoTom
-                  ? new Date(brevutfallOgEtterbetaling.etterbetaling?.datoTom)
-                  : undefined
-              }
-              onChange={(date) =>
-                setBrevutfallOgEtterbetaling({
-                  ...brevutfallOgEtterbetaling,
-                  etterbetaling: {
-                    ...brevutfallOgEtterbetaling.etterbetaling,
-                    datoTom: date ? formatISO(lastDayOfMonth(date), { representation: 'date' }) : undefined,
-                  },
-                })
-              }
-              label="Til og med"
+    <form onSubmit={handleSubmit((data) => submitBrevutfall(data))}>
+      <VStack gap="8">
+        <VStack gap="4">
+          <Controller
```

Comment:
Mulig å lage en egen komponent for ControlledRadio?

Reply:
Yes boss